docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -127,3 +122,47 @@ This has verified service provider initiated SSO. To verify identity provider in
 SSO, navigate to `https://<proxy-address>/enterprise/saml-idp/login/samltest-id`,
 where `samltest-id` is the friendly name of the service provider object created earlier.
 You should be redirected to the same successful login page seen earlier.
+
+
+## Optional: Creating dedicated role to manage Service Provider
+
+For production, we recommend creating a dedicated role to manage Service Provider.
+
+To create dedicated role, first, ensure you are logged into Teleport as a user that has
+permissions to read and modify `saml_idp_service_provider` objects. The default `editor` role
+has access to this already, but in case you are using a more customized configuration,
+create a role called `sp-manager.yaml` with the following contents:
+
+```yaml
+kind: role
+metadata:
+  name: sp-manager
+spec:
+  allow:
+    rules:
+    - resources:
+      - saml_idp_service_provider
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+version: v7
+```
+
+Create the role with `tctl`:
+
+```code
+$ tctl create sp-manager.yaml
+role 'saml-idp-service-provider-manager' has been created
+```
+
+Add the role to your user. Replace `teleport-admin` with your user name:
+
+```code
+$ tctl users update --set-roles $(tctl get users/<Var name="teleport-admin"/> --format=json | \
+jq -r '.[].spec.roles | join(",")'), sp-manager <Var name="teleport-admin"/>
+User teleport-admin has been updated:
+  New roles: auditor,editor,access,sp-manager
+```
```

Comment:
This only works with local users, consider referencing docs/pages/inclues/add-role-to-user.mdx

Reply:
Thanks for that. I didn't figure we have that in includes. Updated in https://github.com/gravitational/teleport/pull/36467/commits/946f3ed6d5c1c5779f9bafc12252e6c208a29dc6 along with your other suggestions above.